roles/satellite-receptor/defaults/main.yml

```diff
@@ -1,15 +1,13 @@
 ---
+install_receptor_from_rpm: yes
 satellite_url: "https://satellite.example.com/"
 satellite_user: "fifi_user"
 satellite_password: "0p3ns3s4m3!"
 satellite_ca_file: "/etc/pki/tls/certs/ca-bundle.crt"
 satellite_validate_certs: yes
 receptor_config_dir: "/etc/receptor"
-# TODO: Find a better place for this, something like /var/lib/receptor
-receptor_data_dir: /tmp
+receptor_data_dir: /var/data/receptor
```

Comment:
do we need to create/setup this from rpm?

Reply:
Currently this role creates it, but having the rpm do that would be nicer

Reply:
I'm going to make a PR against hte spec file.